acceptance-tests.md

```diff
@@ -124,20 +146,25 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. [Click here to visit the Logging Grafana instance](https://logging-grafana.apps.nerc-ocp-infra.rc.fas.harvard.edu).
         2. Click on the “Explore” button which looks like a compass on the left side. At the top of that page, you can select from the dropdown the infrastructure, audit or application logs. You can then use the Log Browser to browse the different labels like kubernetes_host and select a value for a tag to filter in on actual search results. Continue to add more labels and values to narrow in on the logs you wish to see.
         3. You can build custom dashboards and alerts in this Logging Grafana instance.
+    * Acceptance tests:
+        1. [Click here to visit the Logging Grafana instance and find the events in the logs you are looking for](https://logging-grafana.apps.nerc-ocp-infra.rc.fas.harvard.edu).
 3. **Monitoring and logging for the infrastructure hardware and software that is not OpenShift (for example Grafana)**
     * _As a NERC administrator, I should be able to monitor the status of any infrastructure software or hardware that supports operations for the NERC OpenShift environment, even if it is not itself part of OpenShift._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. You can access many metrics for pods of a particular application like grafana or loki.
-        2. [Click here to visit the cpu usage logs for grafana](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate%7Bnamespace%3D%5C%22grafana%5C%22%7D%22%7D%5D).
-        3. [Click here to visit the cpu usage logs for loki](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate%7Bnamespace%3D%5C%22openshift-operators-redhat%5C%22%7D%22%7D%5D).
+    * Acceptance tests:
+        1. [Click here to visit the cpu usage logs for grafana](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate%7Bnamespace%3D%5C%22grafana%5C%22%7D%22%7D%5D).
+        2. [Click here to visit the cpu usage logs for loki](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate%7Bnamespace%3D%5C%22openshift-operators-redhat%5C%22%7D%22%7D%5D).
 
 
 ## Reporting
```

Comment:
Data Management section follows Reporting, but is missing here.  This is basically how you check that archiving is done and that data that is supposed to be available is still available for the required amount of time (60 days or 90 days)

Reply:
Data Management use cases still need to be covered.

Reply:
Data Management added to acceptance tests.